### PR TITLE
Feature/17675 type in location header

### PR DESCRIPTION
**Pull request by crbrox**
Implements #1765 

@kzangeli 
@fgalan

### CODE REVIEW

File: src/lib/serviceRoutinesV2/postEntities.cpp [hunk 1]
```diff
@@ -108,6 +108,14 @@ std::string postEntities
   if (rhttpcode == SccOk || rhttpcode == SccNone)
```
Comment (fgalan) [inline src/lib/serviceRoutinesV2/postEntities.cpp]: Suggested CNR regarding this PR:

"Add: ?type URL parameter in Location header upon entity creation in NGSIv2 (Issue #1765)"

File: test/functionalTest/cases/0864_filter_string_scope/filter_string_scope.test [hunk 2]
```diff
@@ -134,7 +134,7 @@ echo
 =======================================================
 HTTP/1.1 201 Created
 Content-Length: 0
-Location: /v2/entities/E1
+Location: /v2/entities/E1?type=T
```
Comment (fgalan) [inline test/functionalTest/cases/0864_filter_string_scope/filter_string_scope.test]: The "Location: ..." examples in the NGSIv2 .apib should be also adjusted in a similar way.

**Comment by kzangeli** [comment]
+1 on the comments by @fgalan.
For everything else, LGTM

File: src/lib/serviceRoutinesV2/postEntities.cpp (hunk 1 above)
Comment (crbrox) [inline src/lib/serviceRoutinesV2/postEntities.cpp]: Fixed f65f415

File: test/functionalTest/cases/0864_filter_string_scope/filter_string_scope.test (hunk 2 above)
Comment (crbrox) [inline test/functionalTest/cases/0864_filter_string_scope/filter_string_scope.test]: Fixed 1cfd803

**Comment by fgalan** [comment]
LGTM

**Pull request merged**